unittests/backends/ionq/IonQTester.cpp

```diff
@@ -0,0 +1,201 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
+
+#include "CUDAQTestUtils.h"
+#include "common/FmtCore.h"
+#include "cudaq/algorithm.h"
+#include <fstream>
+#include <gtest/gtest.h>
+#include <stdlib.h>
+
+std::string mockPort = "62454";
```

Comment:
We need to use a different port, since this one is already in use for another mock server...
```suggestion
std::string mockPort = "62455";
```